### PR TITLE
Remove tailwindCSS integrate with CLI

### PR DESCRIPTION
**Pull request by cp-priyanka-g**

### CODE REVIEW

File: vue-frontend/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "sass": "^1.49.9",
     "serve-static": "^1.14.2",
     "swiper": "^8.0.7",
-    "tailwindcss": "^3.0.24",
```
Comment (cp-sumi-k) [inline vue-frontend/package.json:47]: Is this change related to this MR?

Comment (cp-priyanka-g) [reply vue-frontend/package.json:47]: yes

**Review by cp-sumi-k — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]
